ResearchKit/Common/ORKStepViewController.m

```diff
@@ -140,12 +139,13 @@ - (void)stepDidChange {
 - (void)viewWillAppear:(BOOL)animated {
     [super viewWillAppear:animated];
     ORK_Log_Debug(@"%@", self);
+    
     [self setupButtons];
```

Comment:
Can you add some documentation for why this is required at here? In case we forget next time:)